backends/etcdv3/client.go

```diff
@@ -119,7 +120,7 @@ func (c *Client) WatchPrefix(prefix string, keys []string, waitIndex uint64, sto
 
 	for wresp := range rch {
 		for _, ev := range wresp.Events {
-			fmt.Println(string(ev.Kv.Key))
+			log.Debug(fmt.Sprintf("Key updated %s", string(ev.Kv.Key)))
```

Comment:
`Log.Debug` does Sprintf for you, so it's redundant here.
https://github.com/kelseyhightower/confd/blob/master/log/log.go#L51-L54

Reply:
Sorry about that, I'll amend it.

Reply:
@okushchenko done, PTAL